src/components/Releases/ReleasesListHeader.tsx

```diff
@@ -3,10 +3,13 @@ import { HeaderFunc } from '../../shared/components/table/Table';
 
 export const releasesTableColumnClasses = {
   name: 'pf-m-width-30 pf-m-width-20-on-xl wrap-column',
-  created: 'pf-m-width-30 pf-m-width-20-on-xl',
+  created: 'pf-m-width-20 pf-m-width-20-on-xl',
+  duration: 'pf-m-hidden pf-m-width-20 pf-m-visible-on-xl',
   status: 'pf-m-width-20',
   releasePlan: 'pf-m-width-25',
   releaseSnapshot: 'pf-m-hidden pf-m-width-25 pf-m-visible-on-xl',
+  managedPipelineRun: 'pf-m-hidden pf-m-width-25 pf-m-visible-on-xl',
+  tenantPipelineRun: 'pf-m-hidden pf-m-width-25 pf-m-visible-on-xl',
   kebab: 'pf-v5-c-table__action',
```

Comment:
Total width of all `pf-m-width-<number>` should be 100, right now it's adding up to 190 which can cause table resizing issues on different viewports.

Reply:
Ack thanks - will fix!

Reply:
Ok, I think I've fixed it.

I put some extra whitespace in to make it easier (for me) to visually confirm that the numbers make sense.

I don't see that style elsewhere in the repo though. Is that kind of whitespace acceptable? If not, I can remove it.

Reply:
@sahil143 I rebased on `main`.

Reply:
I was missing an update to `src/types/release.ts`, fixed just now.